Mlem/Views/Shared/Posts/Feed Post.swift

```diff
@@ -288,6 +288,28 @@ struct FeedPost: View {
             )
         }
     }
+    
+    func blockCommunity() async {
+        do {
+            let blocked = try await Mlem.blockCommunity(
+                account: appState.currentActiveAccount,
+                community: postView.community,
+                blocked: true
+            )
+            if blocked {
+                postTracker.removePosts(from: postView.creator.id)
```

Comment:
This will remove posts by the creator of the blocked post rather than from the community.

In `PostTracker`, rename `removePosts(from personId: Int)` to `removeUserPosts(from personId: Int)` and create a function `removeCommunityPosts(from communityId: Int)` that removes by community, then use that here.

Reply:
Good catch, done! And thanks for the welcome :)